test/ffmpeg-vaapi/encode/encoder.py

```diff
@@ -46,6 +46,8 @@ def gen_output_opts(self):
         opts += " -global_quality {quality}"
       else:
         opts += " -quality {quality}"
+    if vars(self).get("icq_quality", None) is not None:
+      opts += " -quality {icq_quality}"
```

Comment:
I thought `-quality` in ffmpeg-vaapi is only used to set the target-usage?  How can it influence icq_quality?

AFAICT, you need to set `-global_quality` (https://github.com/intel-media-ci/ffmpeg/blob/b414cff6304071d5ef1e2b0a1b407e4fe3f58092/libavcodec/vaapi_encode.c#L1523).